README.md

```diff
@@ -110,9 +110,9 @@ curl --request POST 'http://localhost:4000/api/checks/executions/start' \
 
 In order to get detailed information for an execution, see [Getting Execution details](#getting-execution-details).
 
-> Please note that execution is _eventually started_, meaning that a successful response to the previous API call does not guarantee that the execution is running, but that it has been accepted by the system to start.
+> Please note that execution is _eventually started_, meaning that a successful response the previous API call does not guarantee that the execution is running, but that it has been accepted by the system to start.
```

Comment:
I also wouldn't remove the `to` from here. Maybe I'd further clarify and substitute "...a successful response to the previous API call..." with something that specifically mentions the checks execution request call.